src/auth/models/Auth.class.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
```

Comment:
The header part should be
```
<?php

/**
 * Syracuse
 *
 * @version     1.0 Beta 1
 * @author      Aeros Development
 * @copyright   2017-2018 Syracuse
 * @since       1.0 Beta 1
 *
 * @license     MIT
 */
```